cmd/ltctl/reset.go

```diff
@@ -79,23 +79,28 @@ func RunResetCmdF(cmd *cobra.Command, args []string) error {
 		clients []*ssh.Client
 	}{
 		{
-			msg:     "Resetting database",
+			msg:     "Shutting down MM server on primary...",
+			value:   "sudo systemctl stop mattermost",
+			clients: []*ssh.Client{appClients[0]},
+		},
```

Comment:
Do we need to do it on primary only?

Reply:
Yes, because we are running the `reset` cmd on primary only